### PR TITLE
Cisco umbrella modeling rules

### PR DESCRIPTION
**Pull request by yasta5**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-6065

## Description
This pack contains modeling and parsing rules for cisco umbrella logs:

- DNS
- Proxy
- Admin Audit logs

### CODE REVIEW

File: Packs/Cisco-umbrella-cloud-security/README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+1. A login to Amazon AWS service
+2. A bucket configured in Amazon S3 to be used for storing logs.
+
+See this[doc](https://docs.umbrella.com/deployment-umbrella/docs/setting-up-an-amazon-s3-bucket#prerequisites) for further instructions on how to create a S3 bucket, and how to configure it for your needs.
+
+**On Cisco Umbrella cloud security**
+
+1. Navigate to **Admin** -> **Log Management** and select **Use your company-managed Amazon S3 bucket**.
+2. In the **Bucket Name** put the bucket name you created in Amazon S3 and click **Verify**.
+3. After Cisco Umbrella verifies your bucket, it saves file called **README_FROM_UMBRELLA.txt** to it.
```
Comment (ShirleyDenkberg) [inline Packs/Cisco-umbrella-cloud-security/README.md:20]:
```suggestion
3. After Cisco Umbrella verifies your bucket, it saves a file called **README_FROM_UMBRELLA.txt** in the bucket. 
```

File: Packs/Cisco-umbrella-cloud-security/README.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+- auditlogs
+- dnslogs
+- proxylogs
+
+Note:
+Make sure that the Log schema version is configure to **v8**.
+More information can be found [here](https://docs.umbrella.com/deployment-umbrella/docs/log-formats-and-versioning)
+
+
+**On XSIAM:**
```
Comment (ShirleyDenkberg) [inline Packs/Cisco-umbrella-cloud-security/README.md:35]:
```suggestion
**On Cortex XSIAM:**
```

File: Packs/Cisco-umbrella-cloud-security/README.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+1. Navigate to **Settings** -> **Data Sources** -> **Add Data Source**.
```
Comment (ShirleyDenkberg) [inline Packs/Cisco-umbrella-cloud-security/README.md:37]:
```suggestion
1. Navigate to **Settings** > **Data Sources** > **Add Data Source**.
```

**Comment by ShirleyDenkberg** [comment]
@yasta5 Doc review completed.

**Review by eepstain — APPROVED** [review]

**Pull request merged**